example/demo/README.md

```diff
@@ -29,6 +29,9 @@ docker run -d -p 6379:6379 redis:latest
 
 ### 3. do hybrid compilation
 
+First, please make sure your `go` version is >= `1.22`, which is required
```

Comment:
Better to refer to https://github.com/alibaba/opentelemetry-go-auto-instrumentation/blob/main/docs/compatibility.md

Reply:
Done.